pkgs/by-name/ri/ride/package.nix

```diff
@@ -79,11 +88,13 @@ buildNpmPackage rec {
     popd
   '';
 
-  nativeBuildInputs = [
-    zip
-    makeWrapper
-    copyDesktopItems
-  ];
+  nativeBuildInputs =
+    [
+      zip
+      makeWrapper
+    ]
+    ++ lib.optionals (!stdenv.isDarwin) [ copyDesktopItems ]
+    ++ lib.optionals stdenv.isDarwin [ darwin.cctools ];
```

Comment:
```suggestion
  nativeBuildInputs = [
    zip
    makeWrapper
  ] ++ lib.optionals stdenv.isLinux [ copyDesktopItems ]
    ++ lib.optionals stdenv.isDarwin [ darwin.cctools ];
```

Reply:
Should I replace `!stdenv.isDarwin` with `stdenv.isLinux` everywhere?

Also, the formatting is done by `nixfmt-rfc-style`, so I won't change that.